V5 Release note - Productivity part (#879)
akeneo/pim-helpcenter: content/updates/productivity/4-A-manual-execution-of-asset-naming-conventions.md

# Manual execution of asset naming conventions
::: meta-data type="New" ee-only="true" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention"
:::

> You can now manually execute asset naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after asset uploads or creations.

Reviewer: ```suggestion
You can now manually execute asset naming conventions from the user interface whenever you need it. This new feature can be very useful if your naming convention has changed after assets were uploaded or created.
```

> Asset naming conventions populate asset attributes with values contained in the asset filename or asset code. Prior to this improvement, naming conventions were executed once at the asset creation step.

Reviewer: ```suggestion
Asset naming conventions populate asset attributes with values contained in the asset filename or asset code. Prior to this improvement, naming conventions were executed once, when the asset was created.
```

There are two ways to manually execute a naming convention:

> - for one specific asset, directly in the asset edit form. The action is visible under the `...`.

Reviewer: ```suggestion
- for one specific asset, directly in the asset edit form. The action is visible under the `...` in the top right corner.
```

Reviewer: not sure about the location
